src/DynamoCoreWpf/ViewModels/Watch3D/DynamoGeometryModel3D.cs

```diff
@@ -58,258 +40,5 @@ public bool RequiresPerVertexColoration
             }
             set { SetValue(RequiresPerVertexColorationProperty, value); }
         }
-
-        public override int VertexSizeInBytes
-        {
-            get
-            {
-                return DynamoMeshVertex.SizeInBytes;
-            }
-        }
-
-        protected override void OnRasterStateChanged(int depthBias)
```

Comment:
this also still exists, but no longer has a depthBias parameter.

Reply:
It is not present at all in 2.9.